src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +70,6 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
```

Comment:
Don't forget to write a header comment for `isMutating()`. If not, users of the method as well as those who override this method will not know how exactly the method behaves.